addons/l10n_us_account/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.wise</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <setting id="sepa_payments" position="after">
+                <setting id="wise_payment_settings" string="U.S. Direct Deposit (via Wise)" company_dependent="1" help="Initiate payments in one-click using a Wise Business account"
+                    documentation="/applications/finance/accounting/payables/pay/sepa.html">
```

Comment:
```suggestion
```
I don't see anything related to Wise at the moment. If we have documentation in the future we can definitely add it here.